runner/app/pipelines/audio_to_text.py

```diff
@@ -107,7 +112,7 @@ def __init__(self, model_id: str):
             tokenizer=processor.tokenizer,
             feature_extractor=processor.feature_extractor,
             max_new_tokens=128,
-            device=torch_device,
+            device_map={"", torch_device},
```

Comment:
@RUFFY-369, @eliteprox I had to change this from device to device_maps to ensure all parts of the inference workflow ere on the right device.

Reply:
:eyes: :+1: